README.md

```diff
@@ -3,7 +3,9 @@
 </div>
 
 <a name="readme-top"></a>
+
```

Comment:
I'm going to use this line to request changes about the `N + 1` problems since there are no changes in any controller or view file in this Pull Request

- [ ] Kindly fix `N + 1` problems and follow best practices and optimizations in your queries to the database, refactor your code to render all the posts and all the comments. So, let's say that where you have something like `@posts = @user.posts` would be better to use `@posts = @user.posts.includes(:comments)` and so on with other same queries